mesos/Makefile

```diff
@@ -29,7 +29,7 @@ push: check-version
 mesos: check-version
 	sed "s/VERSION/$(VERSION)/g" dockerfile-templates/$@ > $@
 	if test `echo $(VERSION) | grep rc`; then \
-	  sed -i -e "s/trusty/trusty-testing/g" $@ ; \
+	  sed -i -e "s/{MESOS-TRUSTY}/trusty-testing/g" $@ ; \
```

Comment:
I am wondering if we should rename this to `MESOSPHERE_REPO` instead and then use `sed` to replace it with `trusty` or `trusty-testing`. Any thoughts?